test/functional/test.sh

```diff
@@ -21,6 +21,35 @@ do
     fi
 done
 
+declare -r site_convert="test_site_convert"
+
+# print site name
+echo
+echo "Running $site_convert test"
+
+# convert site
+node ../../index.js init "$site_convert"/non_markbind_site -c
+
+# build site
+node ../../index.js build "$site_convert"/non_markbind_site
+
+# copy generated site
+cp -r "$site_convert"/non_markbind_site/_site "$site_convert"
+
+# run our test script to compare html files
+node testUtil/test.js "$site_convert"
+
+if [ $? -ne 0 ]
+then
+    echo "Test result: $site FAILED"
+    exit 1
+fi
+
+# delete generated files
+rm -rf "$site_convert"/_site
+rm -rf "$site_convert"/non_markbind_site/_markbind "$site_convert"/non_markbind_site/_site
+rm "$site_convert"/non_markbind_site/about.md "$site_convert"/non_markbind_site/index.md "$site_convert"/non_markbind_site/site.json
```

Comment:
It test fails, the generated files will be left in the test site.

Reply:
Pushed a fix for this using `trap` in `test.sh`.